### ISSUE TITLE
Python 3.7 : ModuleNotFoundError: No module named 'distutils.util' using Ubuntu 18.04 image

### ISSUE THREAD

**Issue by wenhuang1003** [deadsnakes/issues#88]
### description

No packege `python3.7-distutils` for Ubuntu bionic. So `setuptools` and `pip` cannot be installed.

#### lsb_release -a

```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.2 LTS
Release:	18.04
Codename:	bionic
```

#### uname -a

```
Linux wenhuang-Precision-Tower-5810 4.18.0-21-generic #22~18.04.1-Ubuntu SMP Thu May 16 15:07:19 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by asottile** [comment]
Did you try installing `python3-distutils`?

**Comment by wenhuang1003** [comment]
> Did you try installing `python3-distutils`?

It's working now! Thank you very much.  So according to [`File list of deb package`](https://packages.ubuntu.com/bionic/all/python3-distutils/filelist), both distutils for 3.6 and 3.7 are installed at the same time.

**Comment by asottile** [comment]
Yeah that package is a little bit weird, glad that fixed it!

**Comment by stonecauldron** [comment]
For anyone who might have faced the same problem as I did:

I was having the same error as this issue showed, however, I had already installed `python3-distutils` so I though it came from somewhere else. To solve the problem, I actually had to reinstall `python3-distutils` after installing `python3.7` i.e. `sudo apt install --reinstall python3-distutils`. I imagine this comes from the fact that the `python3-distutils` package does not install the python3.7 related files if it does not see a python3.7 installation.
